Kickstarter-iOS/Library/SharedFunctions.swift

```diff
@@ -1,6 +1,38 @@
 import Library
 import UIKit
 
+// MARK: - Haptic feedback
+
+func generateImpactFeedback(
+  feedbackGenerator: UIImpactFeedbackGeneratorType = UIImpactFeedbackGenerator(style: .light)
+) {
+  feedbackGenerator.prepare()
+  feedbackGenerator.impactOccurred()
+}
+
+func generateNotificationSuccessFeedback(
```

Comment:
Any concern with these not being prefixed `ksr_`?

Reply:
Since those are not extension functions or anything like that I went with simplicity